lib/server/utilities/findAndCountAllToHttpView.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * @typedef CountedData
+ * @property {Number} count
+ * @property {Object[]} rows
```

Comment:
Here you should use a template instead of Object[]. I would suggest to name it `CountedRows` because you have `count` and `rows`

Reply:
yes refactored, but ... as it's not belong to scope of this PR I will add in another one